packages/engine/Source/Core/CircleGeometry.js

```diff
@@ -90,7 +90,7 @@ const scratchEllipseGeometry = new EllipseGeometry({
 const scratchOptions = {
   center: new Cartesian3(),
   radius: undefined,
-  ellipsoid: Ellipsoid.clone(Ellipsoid.UNIT_SPHERE),
+  ellipsoid: Ellipsoid.clone(Ellipsoid.default),
```

Comment:
Was this intentional or something that just didn't cleaned up?  This would change the behavior for  any callers not passing in a ellipsoid.